### PR TITLE
#86 - Add a wallet Top-up functionality

### LINKED ISSUE #86: Add an option to top up the wallet
## Description:
Give an advanced option to top up the newly created wallet by transferring the value from the existing wallets.

- Add a checkbox called `Top up the wallet`
- Dynamic textbox for value
- Dynamic dropdown menu to select the existing use wallets
- Integrate the `send` (wallet), and `get` (safe) services after creating the wallet

### PR DESCRIPTION
**Pull request by Koushith**
## Description

Give an advanced option to top up the newly created wallet by transferring the value from the existing wallets.

- Add a checkbox called `Top up the wallet`
- Dynamic textbox for value
- Dynamic dropdown menu to select the existing use wallets
- Integrate the `send` (wallet), and `get` (safe) services after creating the wallet
- Form Validation

## Screenshot
image
image

### CODE REVIEW

File: src/screens/create-wallet/create-wallet.screen.tsx [hunk 1]
```diff
@@ -18,82 +29,116 @@ import {
...
+  const [validator, setValidator] = useState(false);
+
+  const getAllWalletsAndFetchBalance = async () => {
+    const wallets = await accountStore.safientUser?.safes.map((safes) => ({
+      label: safes.safeName,
+      value: safes.safeId,
+    }));
+
+    setSelectWallet(wallets);
+    setBalanceLoadder(true);
```
Comment (koshikraj) [inline src/screens/create-wallet/create-wallet.screen.tsx]: Typo of word `Loadder`

File: src/screens/create-wallet/create-wallet.screen.tsx [hunk 2]
```diff
@@ -18,82 +29,116 @@ import {
...
+    safeStore.setRole('creator');
+    const safeData = await safeService.recover(options, 'creator');
```
Comment (koshikraj) [inline src/screens/create-wallet/create-wallet.screen.tsx:75]: Separate the wallet balance fetch functionality and call it after the wallet is selected.

**Comment by Koushith** [comment]
- [x] Fixed all typos
- [x] Separated out ```getAllWallets``` and ```loadBalance``` functions and removed from useEffect.

**Comment by Koushith** [comment]
### Changes

- [x] Replaced old logo.
- [x] Mobile Responsiveness.

**Review by koshikraj — APPROVED** [review]
LGTM. Fix the image size on the landing page. We can merge this.

**Comment by Koushith** [comment]
- [x] Fixed the preview image size.

**Pull request merged**